IFC4x3/Properties/s/SignalHasWalkway_2gs5FGmkn2IRImOUM1ogyr/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SignalHasWalkway_2gs5FGmkn2IRImOUM1ogyr" Name="SignalHasWalkway" UniqueId="aad853d0-c2ec-4249-b4b0-61e581caaf35" PrimaryDataType="IfcBoolean" />
```

Comment:
Could just model the walkway.

Reply:
This property is removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/179/commits/727adb02cf2cfa0bdaa285169fb36fc8ca4ff636